app/javascript/components/payment-error-button.vue

```diff
@@ -159,6 +162,18 @@ export default {
     getUsers() {
       return invoiceApi.getSlackUsers();
     },
+    notifySellersOfDebt(products) {
+      const message = `${this.message.displayNameNormalized} acaba de fiar un producto tuyo!`;
```

Comment:
quizás una variable `displayName` para  `this.message.displayNameNormalized` que lo usas varias veces